charts/miggo-operator/values.yaml

```diff
@@ -106,6 +111,34 @@ updater:
     secretAccessKey: ""
     region: ""
 
+k8sread:
+  enabled: false
+  stdout: false
+  clusterName: "kubernetes-cluster"
```

Comment:
Cant we get the cluster name inside the k8sread?

Reply:
what is a cluster name? i'm not sure there is a kubernetes thing that represent cluster name